tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -12781,63 +12775,10 @@ Decl* Sema::getActiveHLSLBuffer() const
   return HLSLBuffers.empty() ? nullptr : HLSLBuffers.back();
 }
 
-Decl *Sema::ActOnHLSLBufferView(Scope *bufferScope, SourceLocation KwLoc,
```

Comment:
Is this just removing unused code which is not related to the struct change?

Reply:
Yes, unused code that had a change related to this struct change, which I tried and failed to add a test for because the code was dead.